Source/WebCore/rendering/RenderObject.cpp

```diff
@@ -2642,6 +2642,17 @@ static void adjustTextDirectionForCoalescedGeometries(const SimpleRange& range,
     }
 }
 
+static bool isDescendantOutOfFlow(RenderObject* descendant, const RenderObject* ancestor)
+{
+    for (; descendant; descendant = descendant->containingBlock()) {
```

Comment:
This is slightly confusing as you walk **_up_** on the **_ancestor_** chain and yet the variable is called **_descendant_**.

Reply:
Is this a containing **_block_** or a container based helper? i.e. is it okay to skip over inflow positioned inline boxes?

Reply:
Yes, true, skipping inline boxes is wrong, I will use `parent()` here, and add a test for it.